test/Microsoft.AspNet.Mvc.FunctionalTests/TryValidateModelTest.cs

```diff
@@ -0,0 +1,66 @@
+﻿// Copyright (c) Microsoft Open Technologies, Inc. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
+using System;
+using System.Net;
+using System.Threading.Tasks;
+using Microsoft.AspNet.Builder;
+using Microsoft.AspNet.TestHost;
+using Xunit;
+
+namespace Microsoft.AspNet.Mvc.FunctionalTests
+{
+    public class TryValidateModelTest
```

Comment:
sorry, I meant more than one validation error per property.  should also have said this was a nit.